cirq-core/cirq/circuits/circuit_operation_test.py

```diff
@@ -742,6 +746,35 @@ def test_decompose_repeated_nested_measurements():
     assert cirq.Circuit(cirq.decompose(op3)) == expected_circuit
     assert cirq.measurement_keys(expected_circuit) == set(expected_measurement_keys_in_order)
 
+    # Verify that mapped_circuit gives the same operations.
+    assert list(op3.mapped_circuit(deep=True).all_operations()) == list(
+        expected_circuit.all_operations()
+    )
```

Comment:
Here, too, does checking circuit equality not work?

Reply:
Done.